### PR TITLE
Implemented new endpoint for updating Hendelser and set eksportert flag

### PR DESCRIPTION
**Pull request by teddytwin**

### CODE REVIEW

File: src/test/kotlin/no/nav/dsop/kontroll/services/HendelseServiceSpek.kt [hunk 1]
```diff
@@ -159,6 +164,53 @@ object HendelseServiceSpek : Spek({
...
+        describe("operations on eksportere transaksjoner") {
+            val kontrollobjektId = "12345678"
+            val saksnummer = "12345"
+            val utloserVarsling = true
+            val fraDato = "2019-01-01"
+            val tilDato = "2019-01-02"
+            val nodvendighetsvurdering = "test"
+            val metadata = objectMapper.readTree("""{"test": "value"}""")
+            val hendelsestype = Hendelsestype.APPLIKASJON_EKSPORTER_TRANSAKSJONER
+            val hendelse = HendelsesRequest.Sak(
```
Comment (tronghn) [inline src/test/kotlin/no/nav/dsop/kontroll/services/HendelseServiceSpek.kt]: Unused?

File: src/main/kotlin/no/nav/dsop/kontroll/service/hendelse/Models.kt [hunk 2]
```diff
@@ -10,6 +10,30 @@ enum class Hendelsestype {
...
+data class EksportertRequest(
+    val hendelsestype: Hendelsestype,
+    val kontrollobjektId: String,
+    val fraDato: String,
+    val tilDato: String,
+    val nodvendighetsvurdering: String
+)
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class Hendelse(
```
Comment (tronghn) [inline src/main/kotlin/no/nav/dsop/kontroll/service/hendelse/Models.kt]: Is this equivalent to `HendelsesResponse` other than the `saksbehandler` field being a string? Why not use that one instead?

File: src/main/kotlin/no/nav/dsop/kontroll/service/hendelse/HendelseService.kt [hunk 3]
```diff
@@ -82,6 +85,40 @@ class HendelseService {
...
+                            (Hendelser.fraDato eq request.fraDato) and
+                            (Hendelser.tilDato eq request.tilDato) and
+                            (Hendelser.kontrollobjektId eq request.kontrollobjektId) and (Hendelser.saksnummer eq saksnummer)
+                }
+                        .mapNotNull(ResultRow::toHendelse)
+                        .map { hendelsesResponse -> hendelsesResponse.id }
+
+                ids.forEach { id ->
+                    Hendelser.update({ Hendelser.id eq id }) {
+                        it[metadata] = objectMapper.writeValueAsString(TransactionHendelseMetadata(
```
Comment (tronghn) [inline src/main/kotlin/no/nav/dsop/kontroll/service/hendelse/HendelseService.kt]: We're replacing the metadata field's content entirely here. Is this intended? (cc @henninghaakonsen)

Comment (henninghaakonsen) [reply src/main/kotlin/no/nav/dsop/kontroll/service/hendelse/HendelseService.kt]: @tronghn, no just update the "eksportert" flag.

**Comment by tronghn** [comment]
I've updated the branch after talking with @henninghaakonsen. The operation should update *all* hendelser matching the given input, and add/update a field called `"eksportert"` to `true` in the metadata object.

The currently supported hendelsestypes for this endpoint are `BANK_INNHENT_GRUNNLAGSDATA, BANK_INNHENT_TRANSAKSJONER` and can be expanded later on when needed.

**Pull request merged**